src/controller/luaopen_milk_controller.cpp

```diff
@@ -40,11 +40,20 @@ static int lua_is_button_released(lua_State* L)
 	return 1;
 }
 
+static int lua_rumble(lua_State* L)
```

Comment:
We should make duration an optional param. and default it to rumble until stopped using a ```stop_rumble``` method. check out ```luaL_optinteger``` 

It'll make the rumble functionality a bit less restrictive. I might want to begin a rumble and then stop it on a callback opposed to making it rumble for 2 seconds. But i can also only make it rumble for 2 seconds if i need to.